app/wyzebridge/wyze_api.py

```diff
@@ -393,6 +404,26 @@ def clear_cache(self, name: Optional[str] = None):
             for token_file in Path(TOKEN_PATH).glob("*.pickle"):
                 token_file.unlink()
 
+    def setup_mtx_proxy(self, cam_name: str, uri: str) -> bool:
+        if not (cam := self.get_camera(cam_name, True)):
+            return False
+        logger.info(f"🎉 Starting KVS Stream for MTX - {cam.nickname}")
+        kvs_stream: Stream = get_camera_stream(
+            auth_info=self.auth,
+            camera=cam,
+        )
+        kvs_stream.params.signaling_url = urllib.parse.unquote(
+            kvs_stream.params.signaling_url
+        )
+        requests.post(
+            f"http://localhost:8080/websocket/{uri}",
+            json=kvs_stream.params.model_dump(),
+            headers={"Content-Type": "application/json"},
+        )
+        sleep(1)
```

Comment:
Why? 
Not saying these needs to be removed - but does the camera need time to "wake up"? Surely there's a better way to tell if it's woken up instead of using a sleep() however. Being said, IoT devices never cease to amaze me when it comes to the strange ways in which they work :slightly_smiling_face: